apisix/admin/v3_adapter.lua

```diff
@@ -180,9 +181,33 @@ function _M.filter(body)
     end
 
     local args = request.get_uri_args()
+    local processed_body = deepcopy(body)
```

Comment:
Why do we need to copy the body?

Reply:
OK, I got it now.

Reply:
@spacewander In the comment above, @soulbird  points out that we need to copy the body to avoid logical and coding problems.
https://github.com/apache/apisix/pull/7778#discussion_r957974650

Do you think we need it?